README.md

```diff
@@ -19,18 +19,19 @@ See [json-schema](http://json-schema.org/) for more details.
 #### [`Composer`](https://github.com/composer/composer) (*will use the Composer ClassLoader*)
 
     $ wget http://getcomposer.org/composer.phar
-    $ php composer.phar require justinrainbow/json-schema:~2.0
+    $ php composer.phar require --no-update justinrainbow/json-schema
+    $ composer.phar update
 
 ## Usage
 
 ```php
 <?php
 
-$data = json_decode(file_get_contents('data.json'));
+$data = json_decode(file_get_contents('data.json')); //NOTE: this needs to be an object, not an array!
 
 // Validate
 $validator = new JsonSchema\Validator;
-$validator->check($data, (object)['$ref' => 'file://' . realpath('schema.json')]);
+$validator->check($data, (object)['$ref' => 'file://' . realpath('schema.json')]); //either provide PHY schema (after json_decode) or just point out to the correct schema on the local filesystem
```

Comment:
PHY - Physical. Might be misleading, sorry. The point was - either provide object after JSON decode, either provide a link to your schema on the filesystem.